docs/sources/flow/when-to-use-flow/opentelemetry-collector.md

```diff
@@ -0,0 +1,109 @@
+---
+description: Learn how to Flow compares to the OpenTelemetry Collector
+menuTitle: OpenTelemetry Collector
+title: OpenTelemetry Collector
+weight: 100
+---
+
+# Comparing Flow with the OpenTelemetry Collector
+
+## Which one of the two should you use? 
+
+Flow may be a better choice if:
+* You prefer River syntax over YAML.
+* You would like to collect "profile" telemetry - Flow can do this via Pyroscope components.
+* You would like to process Prometheus or Loki telemetry without converting it to OTLP.
```

Comment:
Is this the OSS documentation? One extra advantage of Flow is that Grafana Labs offers support for it.